crc/aarch64/crc32_ieee_norm_pmull.h

```diff
@@ -47,11 +47,17 @@
 .equ	br_high_b2, 0x1
 
 	.text
+#ifndef __MACH__
 	.section	.rodata
+#else
+	.section	__DATA,data
+#endif
 	.align	4
 	.set	.lanchor_crc_tab,. + 0
+#ifndef __MACH__
```

Comment:
Is it requirement from Clang? If yes , I think __clang__ is better choice.

Reply:
it seems controlled by llvm::MCAsmInfo::HasDotTypeDotSizeDirective (https://llvm.org/doxygen/classllvm_1_1MCAsmInfo.html#a7c3b8692b75d4808f7c888e61f01e1c8) and it is false in Darwin (https://github.com/llvm/llvm-project/blob/release/9.x/llvm/lib/MC/MCAsmInfoDarwin.cpp#L89).

Well, if ARM Windows support will be added as well, `#if !defined(__MACH__) && !defined(__WIN32__)` is more proper.

Reply:
Yes , you are right. 

But now , there are no enough information about `run time cpu feature detection`  on WIN32 . That's same with Apple .